src/common/api.config.service.ts

```diff
@@ -252,4 +252,21 @@ export class ApiConfigService {
 
     return jwtSecret;
   }
+
+  getExtrasApiUrl(): string {
+    let extrasApiUrl = this.configService.get<string>('urls.extras');
+    if (!extrasApiUrl) {
+      throw new Error('No extras api url present');
+    }
+
+    return extrasApiUrl;
+  }
+
+  getTransactionsScamCheck(): boolean {
+    var transactionsScamCheck = this.configService.get<boolean>('transactionsScamCheck');
+    if (transactionsScamCheck === undefined) {
+      throw new Error('No extras api url present');
```

Comment:
instead of throwing error, a default value can be returned so that this feature can only be disabled (enabled by default)